.github/workflows/ci.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           - 3.7
           - 3.8
           - 3.9
+          - "3.10"
```

Comment:
n00b question: is there a reason why it needs the quotation marks around 3.10 but not the others?

Reply:
If not, Github actions will treat `3.10` as `3.1` based on my prior experience.